dotnet/src/IntegrationTests/testsettings.json

```diff
@@ -75,7 +75,10 @@
     "ConnectionString": "",
     "VectorSearchCollection": "dotnetMSKNearestTest.nearestSearch"
   },
-  "AzureCosmosDB": {
+  "AzureCosmosDBNoSQL": {
```

Comment:
It would be nice to have a powershell script that can create a test database for you using the required settings, and maybe even output the config settings to use here.

Reply:
Makes sense, thanks! Created a task https://github.com/microsoft/semantic-kernel/issues/8101 to handle it separately.